Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: clean install
+
+clean:
+	rm -rf nighresjava/
+	rm -rf cbstools-public/
+	rm -rf imcn-imaging/
+	rm -rf nighres.egg-info/
+	rm -rf nighres_examples/
```

Comment:
This replaces the `clean_install.sh` script.

@piloubazin, should I remove the script or do you prefer having the "yes / no" check from that script?

Reply:
go ahead, this is nicer :)